hugo/content/_index.md

```diff
@@ -3,4 +3,6 @@ type: index
 section: index
 url: /
 title: Machine Learning for Large Scale Code Analysis
+socialImage: img/logo-bg-blue.png
```

Comment:
`logo-bg-blue.png` logo blue is already the default one for the whole landing (according to [`hugo.config.yaml::socialImage`](https://github.com/src-d/landing/blob/master/hugo.config.yaml#L43)), so it's not needed to make it explicit.
```suggestion
```